toolchest_client/files/unpack.py

```diff
@@ -19,6 +20,8 @@ def unpack_files(file_path_to_unpack, output_type):
         return file_path_to_unpack
     elif output_type == OutputType.SAM_FILE:
         return file_path_to_unpack
+    elif output_type == OutputType.S3:
+        return file_path_to_unpack
```

Comment:
Nit: Setting this definition here seems confusing since the purpose of `unpack_files()` is to unpack compressed outputs. `unpack_files()` is only used in the context of `_unpack_output()`, a helper function for `download()` in `api/download.py`, and `download()` gets skipped if the output path is an S3 URI. I would keep the new `OutputType` but remove these two lines for clarity. (Let me know if this breaks tests, though.)

Reply:
This is primarily here for completeness and could be removed (I'm thinking in switch terms and this if chain could be switched to one). However, I do see a possibility where the download prevention is removed if someone wanted to say move a file from our egress bucket to a bucket in their account after a run is finished if they didn't supply an output path at the tool call.